executor/distsql.go

```diff
@@ -68,7 +68,6 @@ var LookupTableTaskChannelSize int32 = 50
 // contains the handles in those index keys.
 type lookupTableTask struct {
 	handles []kv.Handle
-	rowIdx  []int // rowIdx represents the handle index for every row. Only used when keep order.
 	rows    []chunk.Row
```

Comment:
Can you replace ```[]chunk.Row``` with ```[]rowWithID```? Although It needs to change the other code. it is concise.

Reply:
Done. PTAL